examples/simple/README.md

```diff
@@ -0,0 +1,126 @@
+# Simple Example
+
+For this simple example, we will read the contents of the [cereal.csv](cereal.csv)
+file, which is one of the fun beginner datasets provided on [Kaggle](https://www.kaggle.com/crawford/80-cereals/data).
+The script [generate_tabular.py](generate_tabular.py) provided here can
+be run locally to show the table. First, let's look at the input arguments:
+
+```bash
+./generate_tabular.py --help
+usage: generate_tabular.py [-h] [--version] [--no-header] [--delim DELIM]
+                           [input_file]
+
+Pyout Simple Example
+
+positional arguments:
+  input_file     csv file to read into table
+
+optional arguments:
+  -h, --help     show this help message and exit
+  --version      show version of pyout
+  --no-header    data does not have a header
+  --delim DELIM  delimiter to split tabular file (defaults to comma)
```

Comment:
I'm not opposed to it staying as is, but I'm not sure I see the value in having the command-line layer versus just a script that reads in the file explicitly.  Knobs like `--no-header` and `--delim` don't seem too useful in the context an example of using pyout that is crafted around a particular data file.  And I think it comes with the minor downside of making the script longer, making it a bit harder to pull out the important parts about actually using pyout.

As a minor cosmetic note, using `bash` as the language for the blocks of this README leads to some odd highlighting because nearly all of the output isn't actually bash.

Reply:
I think it's okay to have delim and header, because the extra code is very minimal, and it makes it very easy for the user to quickly customize and use a file without a header, or one separated with tabs. For the most part (the majority of users) it can be safely ignored.

Happy to change the highlight language - what would you prefer?

Reply:
I'll just remove it so it's generic.